File: codeforces/24.vanya_and_fence/vaf.py
# Author: José Rodolfo (jric2002)
n, h = list(map(int, input().split(" ")))
min_width = 0
a = [int(height) for height in input().split(" ")]
for i in range(0, n):
  if (a[i] >= 1 and a[i] <= (2 * h)):
    if (a[i] <= h):
      min_width += 1
    else:
      min_width += 2
print(min_width)

File: codeforces/low_difficulty/difficulty_800/24.vanya_and_fence_[677A]/vaf.cpp
#include <iostream>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  unsigned short int n, h, a, min_width = 0;
  cin >> n >> h;
  while (n--) {
    cin >> a;
    if (a >= 1 && a <= (2 * h)) {
      if (a <= h) {
        min_width++;
      }
      else {
        min_width += 2;
      }
    }
  }
  cout << min_width << endl;
  return 0;
}